src/Tizen.Core/Tizen.Core/Task.cs

```diff
@@ -686,10 +632,16 @@ public void Run()
         }
 
         /// <summary>
-        /// Quits the main loop of the TCoreTask.
+        /// Quits the main loop of the task.
         /// </summary>
-        /// <exception cref="ArgumentException">Thrown when the native handle is invalid.</exception>
+        /// <exception cref="ArgumentException">Thrown when the unmanaged handle is invalid.</exception>
         /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation.</exception>
+        /// <remarks>
+        /// This function can be called from any thread.
+        /// It requests the task to finish the current iteration of its loop and stop running.
+        /// All pending events in the event queue will be processed before quitting. Once the task is quit, it cannot be restarted. 
```

Comment:
"task is quit" -> "task is stopped" / "task quits"
cannot be restarted - ok, that's is significant information, but the description of Task states suggests that it returns to "constructed" state and I would assume can be started again, consider clarifying there, either by adding "Finished/Stopped/Terminated" state or add "Constructed state, but cannot be started again"

Reply:
Done. I've changed the description. ("finished")